packages/terra-consumer-nav/src/components/nav-logo/NavLogo.jsx

```diff
@@ -27,12 +27,12 @@ const defaultProps = {
 const cx = classNames.bind(styles);
 
 const NavLogo = ({
-  path,
+  url,
   altText,
   isCard,
   ...customProps
 }) => {
-  const image = <img className={cx('img')} src={path} alt={altText} />;
+  const image = <img className={cx('img')} src={url} alt={altText} />;
```

Comment:
Are you saying remove this completely and let the consumer provide this as a prop to `Nav` or to just export this module for use by the consumer?